src/views/admin/PersonDetailsView.vue

```diff
@@ -1474,18 +1516,58 @@
           </template>
         </v-container>
         <v-divider
+          v-if="authStore.currentUser?.personId !== personStore.currentPerson?.person.id"
           class="border-opacity-100 rounded my-6 mx-4"
           color="#E5EAEF"
           thickness="6"
         ></v-divider>
-        <!-- Delete person -->
         <v-container
-          v-if="authStore.hasPersonenLoeschenPermission"
-          class="person-delete"
+          class="person-lock"
+          v-if="authStore.currentUser?.personId !== personStore.currentPerson?.person.id"
         >
           <v-row class="ml-md-16">
-            <v-col>
+            <v-col
+              v-if="!personStore.loading"
+              data-testid="person-lock-info"
+            >
               <h3 class="subtitle-1">{{ $t('admin.person.status') }}</h3>
+              <v-row class="mt-4 text-body">
+                <v-col
+                  class="text-right"
+                  cols="1"
+                >
+                  <v-icon
+                    v-if="personStore.currentPerson?.person.isLocked"
+                    icon="mdi-lock"
+                    color="red"
+                  ></v-icon>
+                </v-col>
+                <v-col cols="10">
+                  <span>
+                    {{
+                      personStore.currentPerson?.person.isLocked ? t('person.userIsLocked') : t('person.userIsUnlocked')
+                    }}
+                  </span>
+                </v-col>
+              </v-row>
+              <v-row
+                class="mt-0"
+                v-for="{ key, attribute } of getLockInfo"
+                :key="key"
+                cols="10"
+              >
+                <v-col
+                  class="text-right"
+                  cols="4"
+                >
+                  <span class="subtitle-2"> {{ key }}: </span>
+                </v-col>
+                <v-col cols="5">
+                  <span class="text-body">
+                    {{ attribute }}
```

Comment:
In diesem Teil zeigst du die Daten der sperrenden Person an. Die Orgas zeigen wir aber truncated, um Problem bei langen Schulnamen zu vermeiden. Gucke dir die Methode getSskName an in der Komponente. Kannst du die hier irgendwie auch benutzen um den Schulname zu truncaten? on Hover kannst du dann den ganzen Namen zeigen.

Reply:
Bin jetzt den CSS-Weg gegangen, wie es auch in den Views mit den Tabellen gemacht wird.